### PR TITLE
Login screen view updated according to designs

### PR DESCRIPTION
**Pull request by GlebShaucou**
Login screen view updated according to designs

### CODE REVIEW

File: src/screens/Registration/Registration.js [hunk 1]
```diff
@@ -1,6 +1,6 @@
 import React, { useState } from "react";
 import PropTypes from "prop-types";
-import { Text, ScrollView, KeyboardAvoidingView } from "react-native";
+import { Text, ScrollView, KeyboardAvoidingView, View } from "react-native";
```
Comment (AlexeyKaryka) [inline src/screens/Registration/Registration.js]: Why do you import View here if it isn't used?

File: src/screens/Registration/styles.js [hunk 2]
```diff
@@ -5,28 +5,34 @@ export const styles = StyleSheet.create({
   scrollContainer: {
     paddingTop: 20,
     paddingBottom: 12,
-    width: "90%"
+    width: "100%",
```
Comment (AlexeyKaryka) [inline src/screens/Registration/styles.js]: could you please remove trailing comma?

File: src/screens/Login/Login.js [hunk 3]
```diff
@@ -36,42 +36,49 @@ const Login = props => {
   };
 
   return (
-    <KeyboardAvoidingView style={styles.container} behavior="position" enabled>
-      <View style={styles.headerContainer}>
-        <View style={styles.infoImgContainer}>
-          <Image style={styles.infoImg} source={infoImg} />
+    <KeyboardAvoidingView style={styles.container} enabled>
```
Comment (AlexeyKaryka) [inline src/screens/Login/Login.js:60]: Could you please only remain Inputs in KeyboardAvoidingVeiw? when focusing in input all buttons are lifting up to bottom Input in current state

Comment (GlebShaucou) [reply src/screens/Login/Login.js:60]: It does not affects Login screen (inputs remain visible with keyboard), but this is necessary to match with designs.
image

File: src/screens/Registration/Registration.js (hunk 1 above)
Comment (GlebShaucou) [reply src/screens/Registration/Registration.js]: fixed

File: src/screens/Registration/styles.js (hunk 2 above)
Comment (GlebShaucou) [reply src/screens/Registration/styles.js]: fixed

**Comment by AfanasyShevtsov** [comment]
Resolve conflicts, please.

**Pull request merged**